### PR TITLE
Generalized Splitting and Unpolluted doc/ Directory

### PR DESCRIPTION
**Pull request by jyalim**
First I moved the images from the `doc/` directory into an `.assets/` directory so that user's `.vim/doc/` directories don't copy the images over.

Then, I added two new global variables to common.vim: `g:LatexBox_split_type` and `g:LatexBox_split_length`, so users can have horizontal splitting if they so choose. 

Then I modified motion.vim so that `LatexTOC()` uses local variables to nicely determine if it's doing a horizontal or vertical split. This made it easy to adjust the existing code so that no functionality broke.

Then the vim-docs were updated to reflect the two new global variables and what could be done in the `.vimrc` to change the default splitting type.

Finally, the `README` was adjusted to include a screenshot of the horizontally split TOC and to explain that the `.vimrc` needs to set `filetype plugin on` before LaTeX-Box will work, as well as a small note on how there exist configurable global variables within the plugin that affect things such as the split type.

Personally, I think that the TOC feature is more usable and less cluttered with the horizontal split, but this pull respects that the default was originally vertical.

### CODE REVIEW

**Comment by lervag** [comment]
Wow, this looks like a lot of good work, thanks! I'll try to get the time to look into it this and perform a merge during the weekend.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # LaTeX-BoX
-![LaTeX-BoX](https://raw.github.com/LaTeX-Box-Team/LaTeX-Box/master/doc/LaTeX-BoX.png)
+![LaTeX-BoX](https://raw.github.com/hellabyte/LaTeX-Box/master/.assets/LaTeX-BoX.png)
```
Comment (lervag) [inline README.md]: I guess this should be `LaTeX-Box-Team`?

**Comment by lervag** [comment]
Except for the comment to the Readme, I think these changes look very nice.

**Comment by hellabyte** [comment]
Readme reflects file locations post merge now.

**Comment by lervag** [comment]
Great job, thanks!

**Pull request merged**